tests/integration_tests/functional/test_api.py

```diff
@@ -1386,3 +1386,7 @@ def test_negative_snapshot_load_api(microvm_factory):
         )
 
     assert exc_info.value.args[2].headers["deprecation"]
+
+    # The snapshot/memory files above don't exist, but the request is otherwise syntactically valid.
+    # In this case, Firecracker exits.
+    vm.mark_killed()
```

Comment:
nit: I think there are typos in the commit "test: Have tests where Firecracker exists mark VMs as killed":

> Also remove various variants of "wait for Firecracker process to go
> away", as `mark_killed` as a `wait_for_termination` inside of it.

to

> Also remove various variants of "wait for Firecracker process to go
> away", as `mark_killed` _**has**_ a _**`wait_process_termination`**_ inside of it.

Reply:
o, thanks, fixed!

Reply:
well, actually `wait_for_termination` => `wait_process_termination` as well...

Reply:
Ah, sorry, only saw this now :sob: I'll merge it since the CI is green and I don't trust myself to not mess up the approvals in the force push (since I used GitHub's UI rebase)